docs/pages/kubernetes-access/helm/guides/aws.mdx

```diff
@@ -406,7 +406,7 @@ helm --namespace cert-manager uninstall cert-manager
 
 ### Next steps
 
-You can follow our [Getting Started with Teleport guide](../../../getting-started.mdx#step-2-create-a-teleport-user-and-set-up-two-factor-authentication) to finish setting up your
+You can follow our [Getting Started with Teleport guide](../../../getting-started.mdx#step-24-create-a-teleport-user-and-set-up-two-factor-authentication) to finish setting up your
```

Comment:
Is this a typo? Step-24?

Reply:
No the parser renders the final path that way (minus special character).